### ISSUE TITLE
Issue with JSON encoded args in Powershell

### ISSUE THREAD

**Issue by jdetter** [clockworklabs/SpacetimeDB#191]
From the Discord:
```
PS G:\GitHub\spacetime_sandbox\quickstart-chat\server> spacetime call chat-server send_message '["Hello"]'        
Error: Invalid arguments provided for reducer `send_message` for database `chat-server` resolving to address `5ebd1221afb10aaf61d6c8a527413d82`.

The reducer has the following signature:
        send_message(text: String)

Caused by:
    0: Response text: invalid arguments for reducer send_message: [0]: expected value at line 1 column 2
    1: HTTP status client error (400 Bad Request) for url (http://localhost:3000/database/call/5ebd1221afb10aaf61d6c8a527413d82/send_message)
```

**Comment by jsshizhan** [comment]
same issue

**Comment by jdetter** [comment]
This issue is fixed in 0.7, I wrote out a long description but then accidentally pressed the back button while looking at this in the github projects view and apparently it doesn't save. TLDR, this is the new syntax for the call command:

```bash
spacetime call my_module my_func arg0 arg1 arg2
```

So this should alleviate the need for weird quoting situations in powershell, so I would call this issue fixed.
